src/controllers/error.js

```diff
@@ -9,6 +9,7 @@ exports.client = (req, res) => {
 };
 
 exports.server = (err, req, res, next) => {
+  console.log('500err', err);
```

Comment:
Handle errors properly and remove any console.logs

Reply:
done